bin/namespaced-wireguard-vpn-tunnel

```diff
@@ -5,27 +5,30 @@ die() {
     exit 1
 }
 
+arr=($NETNS_NAME)
```

Comment:
Can we use a more descriptive name than `arr`? Maybe `netns_names` like we did in the interface script.